.github/renovate.json

```diff
@@ -74,11 +74,11 @@
     },
     {
       "matchPackagePatterns": [
-        "^com.google.api:gax",
+        "^com.google.cloud:google-cloud-shared-dependencies",
         "^com.google.api.grpc:proto-google-cloud-alloydb-v1beta",
```

Comment:
You can import the alloydb BOM as defined here: https://github.com/googleapis/google-cloud-java/blob/main/java-alloydb/google-cloud-alloydb-bom/pom.xml

This will manage the versions you are importing in the same way shared dependencies is.

Reply:
updated. Thanks!